decls/postcss.js

```diff
@@ -32,3 +32,9 @@ export type postcss$rule = {
   parent: Object,
   nodes: Array<Object>,
 }
+
+export type postcss$Options = {
```

Comment:
To follow the convention of the other types, let's lowercase `options`: `postcss$options`.